src/main/java/teammates/ui/template/FeedbackResponsePersonRow.java

```diff
@@ -11,6 +11,10 @@
     private String personName;
     private List<FeedbackResponseRow> responses;
 
+    public FeedbackResponsePersonRow() {
```

Comment:
Is there a need for this? It does not look used anywhere.

Reply:
will remove this.